using_images/xpaas_images/sso.adoc

```diff
@@ -0,0 +1,166 @@
+. *Install and Deploy Docker Registry* - Install the Docker Registry and then ensure that the Docker Registry is deployed to locally manage images:
++
+----
+$ oadm registry --config=/etc/origin/master/admin.kubeconfig \
+--credentials=/etc/origin/master/openshift-registry.kubeconfig
+----
++
+For more information, see https://docs.openshift.com/enterprise/3.1/install_config/install/docker_registry.html[Deploying a Docker Registry]
+. *Deploy a Router* - For more information, see https://docs.openshift.com/enterprise/3.1/install_config/install/deploy_router.html[Deploying a Router].
+. Ensure that you can run the _oc create_ command with https://docs.openshift.com/enterprise/3.1/architecture/additional_concepts/authorization.html#roles[cluster-admin] privileges.
+
+== Using the Red Hat Single Sign-On Image Streams and Application Templates
+The Red Hat xPaaS middleware images were
+link:../../install_config/imagestreams_templates.html[automatically created during the installation]
+of OpenShift along with the other default image streams and templates.
+
+== Preparing and Deploying the Red Hat Single Sign-On xPaaS Application Templates
+=== Using the OpenShift CLI
+Log in with a user that holds _cluster-admin_ privileges and perform the following in the {project-name} project space:
+
+. Prepare the JBoss EAP and SSO application service accounts and their associated secrets.
++
+----
+oc create -n {project-name} -f {application-templates_file_path}/secrets/eap-app-secret.json
+----
++
+----
+oc create -n {project-name} -f {application-templates_file_path}secrets/sso-app-secret.json
+----
+. Deploy one of the SSO application templates. This example deploys the *_sso70-postgresql_* template, which deploys two SSO pods, one SSO web server and one encrypted SSO web server, and a third pod for the PostgreSQL database.
++
+----
+$ oc process -f {application-templates_file_path}/sso/sso70-postgresql.json | oc create -n {project-name} -f -
+----
+
+=== Using the OpenShift Web Console
+Log in with a user that holds _cluster-admin_ privileges and perform the following in the {project-name} project space:
+
+. Click *Add to project* to list all of the default image streams and templates.
+. Use the *Filter by keyword* search bar to limit the list to those that match _sso_. You may need to click *See all* to show the desired application template.
+. Click an application template to list all of the deployment parameters. These parameters can be configured manually, or can be left as default. 
+. Click *Create* to deploy the application template.
+
+=== Deployment Process
+Once deployed, three pods will be created, two pods for the SSO web servers and one for the database. A readiness check prevents the web server pods from starting until the database pod is running. After the SSO web server pods have started, they can be accessed at their custom configured hostnames, or at the default hostnames:
```

Comment:
"...three pods will be created" should probably have a colon after it and not a comma.